ansible/roles/monitoring-agent/templates/collectd.conf.j2

```diff
@@ -80,6 +80,16 @@ Import "mesos-master"
 
 <Plugin "python">
 ModulePath "{{ collectd_python_plugin_dir }}"
+Import "mesos-master-tasks"
+  <Module "mesos-master-tasks">
+      Host {{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}
```

Comment:
We ran into some issues due to which we had resort to this. It may have been an ansible bug though. I'll keep it as is for now to be consistent across. Perhaps we should re-look.